lib/pure/concurrency/threadpool.nim

```diff
@@ -262,7 +298,12 @@ proc `^`*[T](fv: FlowVar[T]): T =
     deepCopy result, src
   else:
     result = fv.blob
-  finished(fv)
+  finito(fv)
+
+proc read*[T](future: FlowVar[T]): T =
+  ## Implement just enough of the Future api to make FlowVar compatible with the async macro.
+  ## Returns the value of the future.
```

Comment:
This is an implementation detail. The docs should focus on what this proc is, not how it was implemented. You can add a note about it at the bottom, but it shouldn't be the highlight.